internal/controllers/clusterrequest_controller_test.go

```diff
@@ -592,6 +598,9 @@ var _ = Describe("ClusterRequestReconcile", func() {
 						Raw: []byte(testPolicyTemplateInput),
 					},
 				},
+				Timeout: oranv1alpha1.Timeout{
```

Comment:
Could we also verify that the start timestamp is popluated for hw provisioning in the some of testcases in the ClusterRequestReconcile?

Reply:
Yes, added a few Expect() for the the start timestamp.